### ISSUE TITLE
dialog doesn't fit window

### ISSUE THREAD

**Issue by babobski** [Komodo/KomodoEdit#968]
got this bug for a while now, every time i install an addon i get the dialog if i want to restart komodo.
Only the dialog is too large for my window, i always have to drag the window to the left to access the buttons.
![snap 2015-12-17 at 07 50 58](https://cloud.githubusercontent.com/assets/3530262/11863360/9b19d60a-a493-11e5-897c-68f0fccded3a.png)

![snap 2015-12-17 at 07 51 14](https://cloud.githubusercontent.com/assets/3530262/11863364/a022210c-a493-11e5-8910-d3c1ff7d840e.png)

**Comment by Naatan** [comment]
Does this happen when you use the default skin?

**Comment by babobski** [comment]
This also happens with the default skin, but i noticed i don't have this bug at work.
So i tried to debug this, disabling add ons i don't have at work but this didn't work.
It looked to me every time i restarted komodo the dialog was growing.

But I've got it fixed now by reseting the dialog windows size:
![snap 2015-12-17 at 19 23 40](https://cloud.githubusercontent.com/assets/3530262/11878198/8b53f41a-a4f4-11e5-95a9-b612b6edae60.png)
I maximized the window and minified the window, restarted komodo and the dialog window is now at his normal size again.
The change size option in the dialog is not working, can't change size.

As it only affected my home version and i got it fixed, I'm closing this one, if it shows up again you will be hearing from me.
